pt-BR/mail/chrome/messenger/preferences/downloadactions.dtd

```diff
@@ -1,32 +1,31 @@
-<!ENTITY  downloadactionsWindow.title         "Download Actions">
+<!ENTITY  downloadactionsWindow.title         "Ações de Download">
 <!ENTITY  window.width                         "480">
 <!ENTITY  window.height                        "310">
 
-<!ENTITY  fileTypesDescription.label          "Automatically perform the associated Action with each of the following file types:">
+<!ENTITY  fileTypesDescription.label          "Realizar automaticamente a Ação associada com cada um dos tipos de arquivo abaixo:">
 
-<!ENTITY  fileHandlerColumn.label             "Action">
+<!ENTITY  fileHandlerColumn.label             "Ação">
 <!ENTITY  fileHandlerColumn.accesskey         "A">
-<!ENTITY  fileTypeColumn.label                "File Type">
+<!ENTITY  fileTypeColumn.label                "Tipo de arquivo">
 <!ENTITY  fileTypeColumn.accesskey            "T">
-<!ENTITY  fileMIMETypeColumn.label            "MIME Type">
+<!ENTITY  fileMIMETypeColumn.label            "Tipo MIME">
 <!ENTITY  fileMIMETypeColumn.accesskey        "M">
-<!ENTITY  fileExtensionColumn.label           "Extension">
+<!ENTITY  fileExtensionColumn.label           "Extensão">
 <!ENTITY  fileExtensionColumn.accesskey       "E">
 
-<!ENTITY  remove.label                        "Remove Action">
+<!ENTITY  remove.label                        "Remover ação">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY  remove.label                        "Excluir ação">